packages/visx-stats/test/computeStats.test.ts

```diff
@@ -1,22 +1,28 @@
-import { computeStats } from '../src';
+import { computeStats } from "../src";
```

Comment:
I think the addition of double quotes throughout here is failing `prettier` in the build. you can update them manually or run `yarn format` from the monorepo root.

Reply:
Had an issue running `yarn format` due to a `nimbus` driver not found error. Thus, I ended up doing it manually

Reply:
interesting, should work if you run `yarn` in the root to install all deps. in any case glad you got it!